WordPress/gutenberg
# Replace TT1-Blocks with Empty Theme in the wp-env of gutenberg and CI

At this point, TT1-Blocks is starting to be outdated, we should consider removing it from the Gutenberg dev env and choosing a better theme for development.

We also now have 2022 that is present in the default environment now that it's merged in WordPress trunk.

That said, I decided to replace tt1-blocks with emptytheme here because I think it's the best theme for development, it doesn't have theme styles that can impact behavior of blocks... It's also unopinonated about everything. We still have 2022 in the environment if we want to work with a styled theme but for CI, emptytheme is probably a better default.

WDYT?

I also noticed that in our docs, we still recommend tt1-blocks to test FSE, we should probably update this recommendation separately. 

**testing instructions**

 - Reset your wp-env: `npm run wp-env destroy && npm run wp-env start`
 - Make sure you see the "emptytheme" in the available themes under `Appearance`.
